packages/opentelemetry-metrics/src/Handle.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+
+export class BaseHandle {
+  protected _data = 0;
+  protected _disabled: boolean;
+  protected _monotonic: boolean;
+
+  constructor(disabled: boolean, monotonic: boolean) {
+    this._disabled = disabled;
+    this._monotonic = monotonic;
+  }
+
+  protected validateUpdate() {
```

Comment:
Could this be called something like `isEnabled()`? Or what role does it play besides that?